src/importexport/tabledit/internal/importtef.cpp

```diff
@@ -593,6 +594,79 @@ static void setInstrumentIDs(const std::vector<Part*>& parts)
     }
 }
 
+//---------------------------------------------------------
+//   fillGap
+//---------------------------------------------------------
+
+// Fill one gap (tstart - tend) in this track in this measure with rest(s).
+
+static void fillGap(Measure* measure, track_idx_t track, const Fraction& tstart, const Fraction& tend)
```

Comment:
* `Measure` has a method `fillGap`, maybe you can use that, but you do need to add a parameter to make the rests invisible
* If you prefer using a dedicated static function, then you could perhaps still benefit from the `toRhythmicDurationList` utility, or maybe just `toDurationList`, both are in `durationtype.h`.

Reply:
I would prefer not to use a dedicated function, but could not find a suitable one ... Note that I do not consider rest visibility important in gaps.